server/models/User.js

```diff
@@ -32,6 +41,10 @@ const UserSchema = new mongoose.Schema({
       ref: 'user',
     },
   ],
+  online: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
Like I mentioned before, let's not use the db to keep track of the online users. Instead we'll do that in memory (ticket #65 )

Reply:
Removed online for User. Going to use socketio to determine if user is online